en/docs/ipaas/ballerina/create-your-first-ballerina-event-triggered-integration.md

```diff
@@ -0,0 +1,235 @@
+# Create Your First Ballerina Event-Triggered Integration
```

Comment:
See general comments. Attached an MD file with the tutorial re-written with changes.